flows/definition/localization.go

```diff
@@ -11,35 +11,21 @@ type itemTranslations map[string][]string
 // languageTranslations map a node uuid to item_translations - say "node1-asdf" to { "text": "je suis francais!" }
 type languageTranslations map[flows.UUID]itemTranslations
 
-func (t *languageTranslations) GetTranslations(uuid flows.UUID, key string, backdown []string) []string {
+func (t *languageTranslations) GetTextArray(uuid flows.UUID, key string) ([]string, bool) {
```

Comment:
I guess as long as we never request translations of empty things... then no. Can change